python/lsst/ts/cbp/mock_server.py

```diff
@@ -140,12 +140,12 @@ async def cmd_loop(self):
         """
         while self.connected:
             self.log.debug("In cmd loop")
-            line = await self.reader.readuntil(b"\r")
+            line = await self.reader.readuntil(b"\r\n")
```

Comment:
Maybe make the terminator a constant that can be loaded here so it only needs to be changed in one place if it ever does. The constant could be in `component.py` and could be exported there (though that's not strictly necessary).